client/go/outline/electron/client_windows.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2025 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/binary"
+	"errors"
+	"net"
+	"syscall"
+
+	"github.com/Jigsaw-Code/outline-apps/client/go/outline"
+	"github.com/Jigsaw-Code/outline-sdk/transport"
+	"golang.org/x/sys/windows"
+)
+
+// Define missing Windows constants
+const IP_UNICAST_IF = 31
+
+func newOutlineClient(transportConfig string, adapterIP string, adapterIndex int) (*outline.Client, error) {
+	if adapterIP == "" || adapterIndex < 0 {
```

Comment:
Name it something like `newOutlineClientWithAdapter`, and remove this check. See comment on main.

Reply:
Updated.